cloud/linode/linode.py

```diff
@@ -446,6 +463,8 @@ def main():
             api_key = dict(),
             name = dict(type='str'),
             plan = dict(type='int'),
+            stack_script_id = dict(type='int'),
+            stack_script_udf_responses = dict(type='str'),
```

Comment:
@bcoca That was my first inclination as well. I decided against that when I set this:

`stack_script_udf_responses = dict(type='json'),`

and got this:

`"implementation error: unknown type json requested for stack_script_udf_responses"`